diff --git a/stage2/01-sys-tweaks/00-packages b/stage2/01-sys-tweaks/00-packages
index d56626b..c9c80a4 100644
--- a/stage2/01-sys-tweaks/00-packages
+++ b/stage2/01-sys-tweaks/00-packages
@@ -1,6 +1,6 @@
 ssh less fbset sudo psmisc strace ed ncdu
 console-setup keyboard-configuration debconf-utils parted
-build-essential manpages-dev bash-completion gdb pkg-config
+bash-completion pkg-config
 python-is-python3
 v4l-utils
 gpiod python3-libgpiod
@@ -9,34 +9,25 @@ python3-rpi-lgpio
 python3-spidev
 python3-smbus2
 avahi-daemon
-lua5.1
-luajit
 ca-certificates curl
 usbutils
 dosfstools
-rpi-swap rpi-loop-utils
+rpi-loop-utils
 raspberrypi-sys-mods
 apt-listchanges
 usb-modeswitch
 libpam-chksshpwd
-rpi-update
 libmtp-runtime
-rsync
 htop
-man-db
 ssh-import-id
 ethtool
-ntfs-3g
 pciutils
 rpi-eeprom
 raspi-utils
-udisks2
 unzip zip p7zip-full
 file
 kms++-utils
 python3-venv
 bluez bluez-firmware
 rpi-keyboard-config
-rpi-keyboard-fw-update
 rpi-usb-gadget modemmanager-
\ No newline at end of file
-rpi-connect-lite
diff --git a/stage2/01-sys-tweaks/00-packages-nr b/stage2/01-sys-tweaks/00-packages-nr
index 15c0f66..e69de29 100644
--- a/stage2/01-sys-tweaks/00-packages-nr
+++ b/stage2/01-sys-tweaks/00-packages-nr
@@ -1,3 +0,0 @@
-cifs-utils
-rpicam-apps-lite
-mkvtoolnix
